js/projects.js

```diff
@@ -12,17 +12,17 @@ var projects = [
   {
     title: "Grandmother Trivia",
     date:"August 2017",
-    image: "images/pic.jpg",
+    image: "images/grandmother.jpg",
     link:"",
     description: "Lorem ipsum dolor sit amet, consectetur adipiscing elit. Etiam blandit, leo lacinia hendrerit consequat, odio libero imperdiet ipsum, sed mattis mauris ipsum sed neque. Maecenas in finibus nunc. Morbi hendrerit nulla vel ornare consequat. Curabitur sed odio mi. Proin vitae mauris ut nulla varius pharetra. Pellentesque vel iaculis massa. Etiam commodo nibh quis nisi luctus suscipit. Donec ac ante purus. Nunc a orci velit. Fusce auctor tristique metus. Sed vel nunc eu turpis viverra tempor. Donec ac lorem volutpat, cursus augue euismod, aliquam ante. Donec sem mi, dictum id lacinia sed, porta quis lorem. Ut in finibus elit.",
-    technologies: "techUsed",
+    technologies: "",
```

Comment:
So you can still use an array here. Then add all the technologies to the <ul> using the jQuery .each() method.

Reply:
You might also be able to use the handlebars each built-in helper, in your template, like this:
```
<ul>
  {{#each technologies}}
    <li>{{this}}</li>
  {{/each}}
</ul>
```
Read the docs http://handlebarsjs.com/builtin_helpers.html